### PR TITLE
[feature](merge-cloud) Support userid in cloud for `internal copy into`

### PR DESCRIPTION
**Pull request by deardeng**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by deardeng** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/CreateUserStmt.java
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/CreateUserStmt.java:1]: we may need some regression test?

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by deardeng** [comment]
run buildall

**Comment by deardeng** [comment]
run buildall

**Comment by deardeng** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**